documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -91,40 +81,33 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
 
     conda activate rmg_env
 
-#. Switch the conda solver to libmamba again, to accelerate any changes you might make to this conda environment in the future::
-
-    conda config --set solver libmamba
-
 #. Compile RMG-Py after activating the conda environment ::
 
     make
 
-#. Modify environment variables. Add RMG-Py to the PYTHONPATH to ensure that you can access RMG modules from any folder.
-   *This is important before the next step in which julia dependencies are installed.*
-   Also, add your RMG-Py folder to PATH to launch ``rmg.py`` from any folder.
+#. **Optional**: add your RMG-Py folder to ``PATH`` to launch ``rmg.py`` from any folder.
 
-   In general, these commands should be placed in the appropriate shell initialization file.
+   In general, this commands should be placed in the appropriate shell initialization file.
    For Linux users using bash (the default on distributions mentioned here), these should be placed in ``~/.bashrc``.
    For MacOS users using bash (default before MacOS Catalina), these should be placed in ``~/.bash_profile``, which you should create if it doesn't exist.
    For MacOS users using zsh (default beginning in MacOS Catalina), these should be placed in ``~/.zshrc``. ::
 
-    export PYTHONPATH=YourFolder/RMG-Py/:$PYTHONPATH
     export PATH=YourFolder/RMG-Py/:$PATH
 
    NOTE: Make sure to change ``YourFolder`` to the path leading to the ``RMG-Py`` code. Not doing so will lead to an error stating that python cannot find the module ``rmgpy``.
 
    Be sure to either close and reopen your terminal to refresh your environment variables (``source ~/.bashrc`` or ``source ~/.zshrc``).
 
-#. Install and Link Julia dependencies: ::
-
-     julia -e 'using Pkg; Pkg.add("PyCall");Pkg.build("PyCall");Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="for_rmg")); using ReactionMechanismSimulator;'
+#. **Optional (Recommended)**: Install and Link Julia dependencies using ``install_rms.sh``. Ensure that you have modified your environment variables as described above, and then run the following: ::
 
-     python -c "import julia; julia.install(); import diffeqpy; diffeqpy.install()"
+     . install_rms.sh
```

Comment:
```suggestion
     source install_rms.sh
```

Doing `. install_rms.sh` on my Mac fails with `.: no such file or directory: install_rms.sh`.

Doing `source install_rms.sh` it at least tries to run the script, and does install pyjuliacall.